pkg/apiserver/statement/queries.go

```diff
@@ -128,35 +128,20 @@ func QueryStmtTypes(db *gorm.DB) (result []string, err error) {
 // endTime: 1586845800
 // schemas: ["tpcc", "test"]
 // stmtTypes: ["select", "update"]
+// fields: ["digest_text", "sum_latency"]
 func QueryStatementsOverview(
 	db *gorm.DB,
 	beginTime, endTime int,
 	schemas, stmtTypes []string,
-	text string) (result []Model, err error) {
-	fields := getAggrFields(
-		"plan_count",
-		"table_names",
-		"schema_name",
-		"digest",
-		"digest_text",
-		"sum_latency",
-		"exec_count",
-		"max_latency",
-		"avg_latency",
-		"min_latency",
-		"avg_mem",
-		"max_mem",
-		"sum_errors",
-		"sum_warnings",
-		"avg_parse_latency",
-		"max_parse_latency",
-		"avg_compile_latency",
-		"max_compile_latency",
-		"avg_cop_process_time",
-		"max_cop_process_time")
+	text string,
+	fields []string,
+) (result []Model, err error) {
+	fields = append(fields, "schema_name", "digest", "sum_latency") // "schema_name", "digest" for group, "sum_latency" for order
+	aggrFields := getAggrFields(fields...)
```

Comment:
Will there be duplicate fields?

Reply:
Yep, but I think it is fine... right? (didn't meet errors when testing)